mocha-casperjs.js

```diff
@@ -47,8 +47,13 @@ module.exports = function (Mocha, casper, utils) {
     })
   })
 
+  function clearSteps() {
+    casper.step = 0
+    casper.steps = []
+  }
```

Comment:
no no you are completely changing the functionality here, and deeply mucking with internal state of casper by clearing all the steps. `failTests` already handles when the steps are behind.